platform/ios/src/MGLAnnotationView.mm

```diff
@@ -146,6 +151,29 @@ - (void)updateScaleTransformForViewingDistance
     }
 }
 
+- (void)setRotatesToMatchCamera:(BOOL)rotatesToMatchCamera
+{
+    if (_rotatesToMatchCamera != rotatesToMatchCamera)
+    {
+        _rotatesToMatchCamera = rotatesToMatchCamera;
+        [self updateRotateTransform];
+    }
+}
+
+- (void)updateRotateTransform
```

Comment:
Right. I was copy-pasting the code from my private repository that was used with the project. Forgot to add the call to `[self updateRotateTransform];` from `setCenter`.